helm/test-values.yaml

```diff
@@ -0,0 +1,249 @@
+# Default values for Wrongecret-ctf-party.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullPolicy: Always
```

Comment:
It will save some time it takes to download, albeit small/negligible. Don't think we incur any (additional) costs. I think it's just a bit nicer to set it to "IfNotPresent" indeed. Setting this to always inflates the stats

Reply:
I agree, done.